### PR TITLE
Update Metafix.java

### PR DESCRIPTION
**Pull request by TobiasNx**
Add signature and description. The fix modul was missing this. @fsteeg I am not sure why this fails?

### CODE REVIEW

File: metafix/src/main/java/org/metafacture/metafix/Metafix.java [hunk 1]
```diff
@@ -57,6 +61,11 @@
  * @author Christoph Böhme (Metamorph)
  * @author Fabian Steeg (Metafix)
  */
+@Description("Applies a fix transformation to the event stream. Fix definition is given in brackets.") // checkstyle-disable-line ClassDataAbstractionCoupling|ClassFanOutComplexity
```
Comment (fsteeg) [inline metafix/src/main/java/org/metafacture/metafix/Metafix.java:64]:
```suggestion
@Description("Applies a fix transformation to the event stream, given as the path to a fix file or the fixes themselves.") // checkstyle-disable-line ClassDataAbstractionCoupling|ClassFanOutComplexity
```

File: metafix/src/main/java/org/metafacture/metafix/Metafix.java [hunk 2]
```diff
@@ -57,6 +61,11 @@
...
+@In(StreamReceiver.class)
+@Out(StreamReceiver.class)
+@FluxCommand("fix")
+
```
Comment (fsteeg) [inline metafix/src/main/java/org/metafacture/metafix/Metafix.java:68]:
```suggestion
```

**Review by fsteeg — APPROVED** [review]
Some minor inline suggestions.

File: metafix/src/main/java/org/metafacture/metafix/Metafix.java [hunk 3]
```diff
@@ -57,6 +61,10 @@
  * @author Christoph Böhme (Metamorph)
  * @author Fabian Steeg (Metafix)
  */
+@Description("Applies a fix transformation to the event stream, given as the path to a fix file or the fixes themselves.") // checkstyle-disable-line ClassDataAbstractionCoupling|ClassFanOutComplexity
+@In(StreamReceiver.class)
+@Out(StreamReceiver.class)
+@FluxCommand("fix")
 public class Metafix implements StreamPipe<StreamReceiver>, Maps { // checkstyle-disable-line ClassDataAbstractionCoupling
```
[the remark this answers is not in the document]
Comment (TobiasNx) [reply metafix/src/main/java/org/metafacture/metafix/Metafix.java:68]: How do I fix this?

[the remark this answers is not in the document]
Comment (blackwinter) [reply metafix/src/main/java/org/metafacture/metafix/Metafix.java:68]: Just delete it.

**Pull request merged**